pkg/apis/vpc/v1alpha1/subnetport_types.go

```diff
@@ -14,6 +14,10 @@ type SubnetPortSpec struct {
 	Subnet string `json:"subnet,omitempty"`
 	// SubnetSet defines the parent SubnetSet name of the SubnetPort.
 	SubnetSet string `json:"subnetSet,omitempty"`
+	// The IP address.
+	IPAddress string `json:"ipAddress,omitempty"`
```

Comment:
Why making these changes?

Reply:
If these are for backup/restore or VM import, let us use annotations, rather than spec fields. @lxiaopei

Reply:
I think for VM import case, when migrating workload from VLAN to VPC, the IP is statically configured in the VM -> then retrieve the IP and MAC from VM tool -> specify the IP and MAC address and update the SubnetPort . But I am not sure if Migration tool will use SubnetPort CR or call NSX API directly.

Reply:
VM import will create SubnetPort CR too. Probably let us use annotations.

Reply:
OK. Will add annotations for IP and MAC Address. From NSX SubnetPort API, it is "address_bindings": [
                {
                    "ip_address": "172.26.2.99",
                    "mac_address": "00:50:56:9f:97:02",
                    "vlan_id": 0
                }
            ] in the request body to specify IP and MAC address.

Reply:
Oh, if NSX can take the inputs too (I did not know that), could we follow the exact NSX API definition (let us always do that and add comments to explain why for the cases we cannot follow the rule), and add the address binding field.

Reply:
thanks Jianjun. I added addressBindings field.